ENCODE_release.py

```diff
@@ -426,17 +492,19 @@ def run_script(self):
         print("Releasenator version " + str(self.releasenator_version))
         for accession in self.ACCESSIONS:
             print("Processing accession: " + accession)
-            expandedDict = encodedcc.get_ENCODE(accession, self.connection)
-            objectStatus = expandedDict.get("status")
-            obj = expandedDict["@type"][0]
-
+            data = encodedcc.get_ENCODE(accession, self.connection)
+            object_status = data.get('status')
+            object_type = data['@type'][0]
+            if self.associated_with_hela_data(object_type, data):
```

Comment:
Inside associated_with_hela_data the names of the variables are changing.  Also, why not just call them 
data, data_status, and data_type here and in all subsequent functions...Whatever they're called, do we need to change them in the function?